packages/monitor-v2/src/llm-bot/OptimisticOracleV2.ts

```diff
@@ -103,3 +140,12 @@ export class OptimisticOracleClientV2 extends OptimisticOracleClient<OptimisticO
     return new OptimisticOracleClientV2(this.provider, requests, fetchedBlockRange);
   }
 }
+
+export class OptimisticOracleClientV2FilterDisputeable
+  implements OptimisticOracleClientFilter<OptimisticOracleRequest, OptimisticOracleRequest> {
+  async filter(optimisticOracleRequests: OptimisticOracleRequest[]): Promise<OptimisticOracleRequest[]> {
+    return optimisticOracleRequests.filter((request) => {
+      return typeof request.disputableUntil == "number" && request.disputableUntil > Date.now() / 1000;
```

Comment:
Does `request.disputableUntil !== undefined` not work here? Just curious about the roundabout syntax